include/tachimawari/joint.hpp

```diff
@@ -35,28 +35,29 @@ class Joint
 
   void set_target_position(float target_position, float speed = 1.0);
   void set_target_position(float present_position, float target_position, float speed);
+  void set_simulator_target_position(float target_position, float speed);
```

Comment:
The `set_target_position` function should not be separated for simulation or not. It should be abstracted enough so it doesn't care whether it will be used on simulation or not.

I think the problem is on the line 89 of the `joint.cpp`:

https://github.com/ichiro-its/tachimawari/blob/b9ce3f20b5679ae615de5e2658c9fd6664002eeb/src/tachimawari/joint.cpp#L89-L90

This class should save position using common format, either in radian or degree, but not in some servo's position representation (in this case, it uses 0-4096 servo's representation of angle position).